doc/netplan-yaml.md

```diff
@@ -1199,41 +1199,41 @@ wpasupplicant installed if you let the `networkd` renderer handle wifi.
 
   - **fail-over-mac-policy** (scalar)
 
-    > Set whether to set all slaves to the same MAC address when adding
+    > Set whether to set all ports to the same MAC address when adding
     > them to the bond, or how else the system should handle MAC addresses.
     > The possible values are `none`, `active`, and `follow`.
 
   - **gratuitous-arp** (scalar)
 
     > Specify how many ARP packets to send after failover. Once a link is
-    > up on a new slave, a notification is sent and possibly repeated if
+    > up on a new port, a notification is sent and possibly repeated if
     > this value is set to a number greater than `1`. The default value
     > is `1` and valid values are between `1` and `255`. This only
     > affects `active-backup` mode.
     >
     > For historical reasons, the misspelling `gratuitious-arp` is also
     > accepted and has the same function.
 
-  - **packets-per-slave** (scalar)
+  - **packets-per-member**, **packets-per-slave** (scalar)
```

Comment:
I'd rather put the "packets-per-slave" title down into the description, e.g. as Alias: **packets-per-slave**